packages/stream_chat_flutter/lib/src/message_list_view/message_list_view.dart

```diff
@@ -852,7 +857,7 @@ class _StreamMessageListViewState extends State<StreamMessageListView> {
               reverse: widget.reverse,
               itemPositionListener: _itemPositionListener.itemPositions,
               messages: messages,
-              dateDividerBuilder: widget.dateDividerBuilder,
+              floatingDateDividerBuilder: widget.floatingDateDividerBuilder,
```

Comment:
Hey @AleksandrLarin-Ardas , this is going to be a breaking change, we don't have to rename the param in `FloatingDateDivider`. We can reuse the same param name.

Also, for backward compatability we should use the previous builder as a fallback.

```suggestion
              dateDividerBuilder: widget.floatingDateDividerBuilder ?? widget.dateDividerBuilder
```